src/boot/bootctl-install.c

```diff
@@ -397,7 +434,7 @@ static int copy_one_file(const char *esp_path, const char *name, bool force) {
 
         /* Note that if this fails we do the second copy anyway, but return this error code,
          * so we stash it away in a separate variable. */
-        ret = copy_file_with_version_check(source_path, dest_path, force);
+        ret = copy_file_with_version_check(source_path, dest_path, force ? COPY_WITH_VERSION_CHECK_FORCE : 0);
```

Comment:
instead of converting "bool force" into a flags field here, just pass in the flags field directly, i.e. change copy_one_file() to take the flags param as last value directly